master/getting-started/kubernetes/installation/hosted/canal/canal-etcd.yaml

```diff
@@ -1,515 +1,4 @@
-        - name: cni-net-dir
-          hostPath:
-            path: /etc/cni/net.d
-        # Used by flannel daemon.
-        - name: run-flannel
-          hostPath:
-            path: /run/flannel
-        - name: resolv
-          hostPath:
-            path: /etc/resolv.conf
-        # Mount in the etcd TLS secrets.
-        - name: etcd-certs
-          secret:
-            secretName: calico-etcd-secrets
-
----
-
-# This manifest deploys a Job which performs one time
-# configuration of Canal.
-apiVersion: batch/v1
-kind: Job
-metadata:
-  name: configure-canal
-  namespace: kube-system
-  labels:
-    k8s-app: canal
-spec:
-  template:
-    metadata:
-      name: configure-canal
-    spec:
-      hostNetwork: true
-      restartPolicy: OnFailure
-      containers:
-        # Writes basic flannel configuration to etcd.
-        - name: configure-flannel
-          image: quay.io/coreos/etcd:v3.1.5
-          command:
-          - "etcdctl"
-          - "--cert-file=/calico-secrets/etcd-cert"
-          - "--key-file=/calico-secrets/etcd-key"
-          - "--ca-file=/calico-secrets/etcd-ca"
-          - "--no-sync"
-          - "set"
-          - "/coreos.com/network/config"
-          - '{ "Network": "10.244.0.0/16", "Backend": {"Type": "vxlan"} }'
-          env:
-            # The location of the etcd cluster.
-            - name: ETCDCTL_PEERS
-              valueFrom:
-                configMapKeyRef:
-                  name: canal-config
-                  key: etcd_endpoints
-            # The location of the {{site.prodname}} etcd cluster.
-            - name: ETCDCTL_CACERT
-              valueFrom:
-                configMapKeyRef:
-                  name: canal-config
-                  key: etcd_ca
-          volumeMounts:
-            # Mount in the etcd TLS secrets.
-            - mountPath: /calico-secrets
-              name: etcd-certs
-      volumes:
-        # Mount in the etcd TLS secrets.
-        - name: etcd-certs
-          secret:
-            secretName: calico-etcd-secrets
-
----
-
-# This manifest deploys the {{site.prodname}} policy controller on Kubernetes.
-# See https://github.com/projectcalico/k8s-policy
-apiVersion: extensions/v1beta1
-kind: Deployment
-metadata:
-  name: calico-kube-controllers
-  namespace: kube-system
-  labels:
-    k8s-app: calico-kube-controllers
-spec:
-  # The policy controller can only have a single active instance.
-  replicas: 1
-  strategy:
-    type: Recreate
-  template:
-    metadata:
-      name: calico-kube-controllers
-      namespace: kube-system
-      labels:
-        k8s-app: calico-kube-controllers
-      annotations:
-        scheduler.alpha.kubernetes.io/critical-pod: ''
-    spec:
-      # The policy controller must run in the host network namespace so that
-      # it isn't governed by policy that would prevent it from working.
-      hostNetwork: true
-      tolerations:
-        # Make sure canal node can be scheduled on all nodes.
-        - effect: NoSchedule
```

Comment:
looks like we lost this toleration in the new canal  manifest. Not sure what our stance is on running kube-controllers on master...

Reply:
Casey's stance is

> ¯\_(ツ)_/¯